docs/dev/MAINTAINERS.mdx

```diff
@@ -0,0 +1,156 @@
+format that includes a **type**, a **scope** and a **subject**:
+
+```
+<type>(<scope>): <subject>
+<BLANK LINE>
+<body>
+<BLANK LINE>
+<footer>
+```
+
+The **header** is mandatory and the **scope** of the header is optional.
+
+Any line of the commit message cannot be longer 100 characters! This allows the message to be easier
+to read on GitHub as well as in various git tools.
+
+The footer should contain a [closing reference to an issue](https://help.github.com/articles/closing-issues-via-commit-messages/) if any.
+
+A coupled of examples are:
+
+```
+docs(changelog): update changelog to beta.5
+```
+
+```
+fix(release): need to depend on latest rxjs and zone.js
+
+The version in our package.json gets copied to the one we publish, and users need the latest of these.
+```
+
+### Types
+
+- **feat**: A new feature
+- **fix**: A bug fix
+
+### Scopes
+
+- **rpc**: Changes related to the core lightning RPC wrapper package
+- **cln_plugin**: Changes related to the core lightning plugin API package
+
+
+### Subject
+
+The subject contains a succinct description of the change:
+
+- use the imperative, present tense: "change" not "changed" nor "changes"
+- don't capitalize the first letter
+- no dot (.) at the end
+
+### Body
+
+You are free to put all the content that you want inside the body, but if you are fixing
+an exception or some wrong behavior you must put the details or stacktrace inside the body to make sure that
+it is indexed from the search engine.
+
+An example of commit body is the following one
+
+```
+checker: fixes overloading operation when the type is optimized
+
+The stacktrace is the following one
+
+} expected `Foo` not `Foo` - both operands must be the same type for operator overloading
+   11 | }
+   12 |
+   13 | fn (_ Foo) == (_ Foo) bool {
+      |                  ~~~
+   14 |     return true
+   15 | }
+
+
+Signed-off-by: Vincenzo Palazzo <vincenzopalazzodev@gmail.com>
+```
+
+## How make the release
+
+This is the most funny part, and also is the most difficult one in a monorepo repository.
+
+In particular, lndart.cln has the possibility to release one single package at the time, or
+all together.
+
+To prepare the release the following steps are required:
+
+- Bump the version number in the package before the release, and the version inside the `changelog.json` in the package root;
+- Generate the changelog related to the package:
+    - `export GITHUB_TOKEN="your_token"`
+    - `make {changelog_client|changelog_flutter|changelog}`, where
+        - `changelog_rpc`: generate the changelog for clightning_rpc;
+        - `changelog_plugin`: generate the changelog for cln_plugin;
+        - `changelog`: generate both changelogs.
+- Make the Github release: To release a single package we need to create a release with the following tag `{package_name}-v{version_number}`, and
+  if we make a release with the tag `v{version_number}` this will release all the packages (useful for a major release of the package).
```

Comment:
```suggestion
- Make the Github release: To release a single package we need to create a release with the following tag `{package_name}-v{version_number}`, and if we make a release with the tag `v{version_number}` this will release all the packages (useful for a major release of the package).
```